packages/mint-components/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - clear success message on re-send
 - \<sqm-portal-reset-password>
   - handle network failure of reset password mutation
+- removed sqm-graphql-client-provider usage from portal templates
```

Comment:
Might need find a better way to tag updates that did not get released with a new versions